pkg/ansibleapp/dev_registry.go

```diff
@@ -1,20 +1,63 @@
 package ansibleapp
 
 import (
+	"bytes"
+	"encoding/json"
 	"fmt"
+	"github.com/op/go-logging"
+	"net/http"
+	"strings"
 )
 
+const APPS_PATH = "/ansibleapps"
+
 type DevRegistry struct {
 	config RegistryConfig
+	log    *logging.Logger
 }
 
-func (r *DevRegistry) Init(config RegistryConfig) error {
+func (r *DevRegistry) Init(config RegistryConfig, log *logging.Logger) error {
+	log.Debug("RHCCRegistry::Init")
 	r.config = config
-	fmt.Printf("DevRegistry::Init with url -> [ %s ] \n", r.config.Url)
+	r.log = log
 	return nil
```

Comment:
I expect real registries to be doing more interesting things with failure possibility; this guys just kind of a test stub for us for now.